firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -3728,14 +3728,14 @@ extension String {
 
 // MARK: - Close tab notifications
     public static let CloseTab_ArrivingNotification_title = MZLocalizedString(
-        key: "CloseTab_ArrivingNotification_title",
+        key: "CloseTab.ArrivingNotification.title.v133",
         tableName: "FxANotification",
-        value: "Firefox tabs closed: %1$@",
+        value: "%1$@ tabs closed: %2$@",
         comment: "Title of notification shown when a remote device has requested to close a number of tabs defined by ($1)")
```

Comment:
Can we update this comment as well?

> Title of notification shown when a remote device has requested to close a number of tabs defined by ($2$@). %1$@ is the name of the app (e.g. Firefox).

Reply:
I will make a PR for this. Thanks @flodolo!